app/tasks/maintenance.php

```diff
@@ -39,6 +39,14 @@ function notifyDeleteAuditLogs(int $interval)
             ]);
         }
 
+        function notifyDeleteConnections() 
+        {
+            Resque::enqueue(Event::DELETE_QUEUE_NAME, Event::DELETE_CLASS_NAME, [
+                'type' => DELETE_TYPE_REALTIME,
+                'timestamp' => time() - 60
```

Comment:
Is this the expected behaviour ? 
"Every _APP_MAINTENANCE_INTERVAL ( or 86400 ) seconds we delete realtime connections older than 60 seconds?  "

Reply:
Yes, this will just clean up unused data that is used to have concurrent connection stats across all containers.